proof_test.go

```diff
@@ -716,16 +716,20 @@ func TestVerifyNamespace_ShortAbsenceProof_Valid(t *testing.T) {
 	qNS := []byte{5} // does not belong to the tree
 	root, err := tree.Root()
 	assert.NoError(t, err)
-
+	// In the following illustration, nodes are suffixed with the range
+	// of leaves they cover, with the upper bound being non-inclusive.
+	// For example, Node3_4 denotes a node that covers the 3rd leaf (excluding the 4th leaf),
+	// while Node4_6 represents the node that covers the 4th and 5th leaves.
+	//
 	//                                       Node_0_8                                  Tree Root
 	//                            /                            \
 	//                        /                                 \
-	//                  Node0_4                             Node4_8                  Non-Leaf Node
+	//                  Node0_4                             Node4_8                    Non-Leaf Node
 	//               /            \                     /                \
 	//             /                \                 /                    \
-	//      Node_0_2            Node_2_4         Node4_6              Node6_8        Non-Leaf Node
+	//      Node_0_2            Node_2_4         Node4_6              Node6_8          Non-Leaf Node
```

Comment:
[optional] some contain a leading `_` and some don't. In my opinion, not a big deal

```suggestion
	//      Node0_2            Node2_4         Node4_6              Node6_8          Non-Leaf Node
```

Reply:
Fixed https://github.com/celestiaorg/nmt/pull/217/commits/1e56c61cce1da94de94841728079635f45db847a